holoviews/plotting/bokeh/plot.py

```diff
@@ -235,24 +263,30 @@ def _create_subplots(self, layout, ranges):
             # Create axes
             kwargs = {}
             if c == 0 and r != 0:
-                kwargs['xaxis'] = 'bottom-bare'
-                kwargs['width'] = 150
+                kwargs['xaxis'] = None
+                kwargs['width'] = self.plot_size+50
             if c != 0 and r == 0:
```

Comment:
Would be much better to query it than parameterize it.  Can it be calculated as outer dimension minus inner dimension, now that the inner dimensions are available in Bokeh JS?